src/dbnode/integration/index_multiple_block_query_test.go

```diff
@@ -23,6 +23,7 @@
 package integration
 
 import (
+	gocontext "context"
```

Comment:
don't have to solve in this PR, but...

we should standardize what we want this important alias to be. i see both `gocontext` and `stdcontext`. My vote is `stdcontext` cause that's typically what you see for things that collide with the std libs. ultimately I don't really care, and just one one consistent alias.

Reply:
Agreed. I'm fine with stdcontext. Let's do in a followup?